### PR TITLE
release 1.6.0

### PR DESCRIPTION
**Pull request by xinlian12**
Release 1.6.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,8 @@
...
-### 1.6.0-beta.1 (Unreleased)
+### 1.6.0 (2022-10-13)
 #### New Features
 * Added bulk support for `CosmosDBSinkConnector`. - [PR 485](https://github.com/microsoft/kafka-connect-cosmosdb/pull/485)
-#### Breaking Changes
-
-#### Key Bug Fixes
 
 #### Other Changes
 * Added gateway mode support for `CosmosDBSinkConnector` and `CosmosDBSourceConnector`. - [PR 490](https://github.com/microsoft/kafka-connect-cosmosdb/pull/490)
```
Comment (kushagraThapar) [inline CHANGELOG.md:8]: I feel this should also be a new feature, thoughts?

**Review by kushagraThapar — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (FabianMeiswinkel) [reply CHANGELOG.md:8]: +1

**Review by FabianMeiswinkel — DISMISSED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (xinlian12) [reply CHANGELOG.md:6]: Make sense - updated

**Review by kushagraThapar — APPROVED** [review]

**Pull request merged**